nixos/modules/services/wayland/swhkd.nix

```
@@ -0,0 +1,28 @@
+{ config, pkgs, lib, ... }:
+
+let
+  inherit (lib) mkIf mkOption mkEnableOption mkPackageOption optional;
+  cfg = config.services.swhkd;
+in {
+  options.services.swhkd = {
+    enable = mkEnableOption (lib.mdDoc "simple wayland hotkey daemon");
+    package = mkPackageOption pkgs "swhkd" {};
+    installManpages = mkOption {
+      type = lib.types.bool;
+      default = true;
+      description = "install swhkd manual pages";
+    };
+    swhkdrc = mkOption {
+      type = lib.types.str;
```

```suggestion
      type = lib.types.lines;
```
we probably want to merge that, don't we?